src/utils/__tests__/LockFile.test.ts

```diff
@@ -39,22 +39,22 @@ describe("LockFile", () => {
                       "20190528171724-test-5",
                     ],
                   }
-            `);
-  });
+            `)
+  })
 
-  test("parse correct file", () => {
+  test('parse correct file', () => {
     const file = dedent(`
   # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
   # Prisma Migrate lockfile v1
-  # You are seeing a conflict here? Just execute \`prisma2 migrate fix\`
+  # You are seeing a conflict here? Just execute \`prisma migrate fix\`
```

Comment:
We should get rid of this line - the command does not even exist :p

Reply:
True! https://github.com/prisma/migrate/commit/ccb9f7bcd397bdbd714f17f38cad3d88c83cff22